test/e2e/e2e_test.go

```diff
@@ -56,6 +58,16 @@ func checkContinuousConnectivity(f *framework.Framework, nodeName, podName, host
 		return
 	}
 
+	// Wait for pod network setup to be almost ready
+	wait.PollImmediate(1*time.Second, 30*time.Second, func() (bool, error) {
+		pod, err := podClient.Get(podName, metav1.GetOptions{})
+		if err != nil {
+			return false, err
+		}
```

Comment:
@dave-tucker should L65 be returning `nil`, since the goal is wait for some time for the pod network to be ready?

Reply:
Same comment at the other places, as well.

Reply:
My thinking was that the pod has already been created and as such a `Get` for that pod shouldn't fail. If it does, then that's probably a bigger problem.
I forgot to actually handle that error though, so I'm glad you commented :smile: 
I've force pushed with err handling - if it works, great!, if not, i'll revert and adopt your suggestion of returning `nil` here